controllers/etcd_controller.go

```diff
@@ -1215,9 +1271,45 @@ func (r *EtcdReconciler) getMapFromEtcd(etcd *druidv1alpha1.Etcd) (map[string]in
 		values["store"] = storeValues
 	}
 
+	if etcd.Spec.Backup.CopySpec != nil {
+		sourceStoreValues, err := getStoreValues(etcd.Spec.Backup.CopySpec.SourceStore)
+		if err != nil {
+			return nil, err
+		}
+		values["sourceStore"] = sourceStoreValues
+		if etcd.Spec.Backup.CopySpec.MaxAge != nil {
+			values["copySpec"] = map[string]interface{}{
+				"maxAge": etcd.Spec.Backup.CopySpec.MaxAge,
+			}
+		}
+		if etcd.Spec.Backup.CopySpec.SnapshotsToCopy != nil {
```

Comment:
I find it a bit confusing that if the second condition is also true, `values["copySpec"]` would be overwritten. I would suggest adding an else or using a switch to make it clear that the 2 conditions are supposed to be mutually exclusive.

Reply:
Before the last changes in the copy operation it was only possible to set one of the those properties, so this is kind of left over